pliers/extractors/image.py

```diff
@@ -119,11 +119,9 @@ def _extract(self, stim):
         return ExtractorResult(values, stim, self, features=feature_names,
                                raw=values)
 
-    def to_df(self, result):
-        n_faces = len(result.raw)
-        cols = [self._feature] if n_faces == 1 else \
-            ['%s_%d' % (self._feature, i) for i in range(1, n_faces + 1)]
-        return pd.Series(result.raw, index=cols).to_frame().T
+    def _to_df(self, result):
+        cols = listify(self._feature)
+        return pd.DataFrame([[r] for r in result.raw], columns=cols)
```

Comment:
Unrelated to this PR, but should these separate features into the coordinates? I.e. the Google face extractor has a feature for `face_boundingPoly_vertex1_X`, `face_boundingPoly_vertex1_Y`, etc.

Reply:
I thought about that, but it seems like overkill for the `FaceRecognitionFaceLocationsExtractor`, which just has 4 values, and is not practical for the `FaceRecognitionFaceLandmarksExtractor`, which can have arbitrary numbers of vertices for different features.

Reply:
Agreed on the face landmarks, since there can be an arbitrary number. I believe expanding out the 128 encodings isn't a crazy idea/would be in line with the `WordEmbeddingsExtractor`